libs/@local/hash-backend-utils/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "description": "HASH backend utilities",
   "license": "AGPL-3.0",
+  "type": "module",
```

Comment:
`jest` is a tricky one, this is already working with the other tests, `jest` becomes `node --experimental-vm-modules $(yarn bin jest)`, then it will work.

(I simply missed `hash-backend-integration` when I was looking through it D:

Reply:
ESM+TS support in Jest is sadly lacking, after looking around the consensus seems to be to move to vitest instead, which is jest compatible. This has been done, see: [d3839be](https://github.com/hashintel/hash/pull/4117/commits/d3839beb67e9f54df54894996d60727768dc6bc9)

The problem is that `setupFiles` apparently can't be ESM+TS modules, which leads to `import.meta` is not defined errors in the environment script. Once I switched to vitest this wasn't a problem and it was simply plug and play.